stripe/src/main/java/com/stripe/android/paymentsheet/ui/BillingAddressView.kt

```diff
@@ -153,7 +146,10 @@ internal class BillingAddressView @JvmOverloads constructor(
     )
 
     init {
-        countryLayout.countryCodeChangeCallback = newCountryCodeCallback
+        countryLayout.countryCodeChangeCallback = {
+            postalCodeView.text = null
```

Comment:
while you are at it, probably update postalCodeView from `StripeEditText` to `PostalCodeEditText` ? I think we can also clean up the additional `PostalCodeConfig` declared inside `BillingAddressView`

`CardFormView` has [similar logic](https://github.com/stripe/stripe-android/blob/a9a0b698cc0ca0e7a64a2f63e8b43a89dd969c0c/stripe/src/main/java/com/stripe/android/view/CardFormView.kt#L213)